trackintel/model/trips.py

```diff
@@ -12,8 +12,11 @@ class TripsAccessor(object):
 
     Requires at least the following columns: 
     ['user_id', 'started_at', 'finished_at', 'origin_staypoint_id', 'destination_staypoint_id']
+    Optional column:
+    ['geom'] 
+    Trips is treated as a GeoDataFrame if a geometry column is provided (containing start and destination points)
 
-    The 'index' of the GeoDataFrame will be treated as unique identifier of the `Trips`
+    The 'index' of the (Geo)DataFrame will be treated as unique identifier of the `Trips`
 
     For several usecases, the following additional columns are required:
     ['context', 'origin_activity', 'destination_activity', 'modes', 'primary_mode', 'tour_id']
```

Comment:
I think the geometry column could be added here. Additionally I would add the sentence "Trips have an optional geometry of type multipoint which describes the start and the end point of the trip" or something similar.

This docstring here is very important because it serves as the main documentation for trips.

Reply:
The sentence I proposed could also be added to "notes"

Reply:
Also changed with the last commits